config/locales/it.yml

```diff
@@ -440,6 +440,7 @@ it:
       nostra %{code_of_conduct_link}. Non sei più bannato.'
   omniauth:
     google: Google
+    github: Github
```

Comment:
So I don't think we need to add another GitHub translation since we already have one at `navigation.github`. In general, we don't want to introduce duplication in our translation files.